### ISSUE TITLE
Start session in user folder instead of /

### ISSUE THREAD

**Issue by tobimori** [matthiaaas/ftp#1]
It would be good to have an option per connection or saved connection where you can choose to start an FTP session in the user folder instead of `/`. This could be activated e.g. for shared hosting.

**Comment by matthiaaas** [comment]
Currently when first connecting to a server, it starts in the root folder (or in ~). If you enter another directory and restart the app, the new directory will automatically be saved.

Not sure if an extra option is necessary?

**Comment by tobimori** [comment]
Yeah, I just noticed that yesterday, but I'd still start in ~ and not in /.
